setup.py

```diff
@@ -1,11 +1,18 @@
 from setuptools import setup
+import sys
 import os
 
 try:
-   import pypandoc
-   long_description = pypandoc.convert('README.md', 'rst')
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
 except (IOError, ImportError):
-   long_description = open('README.md').read()
+    long_description = open('README.md').read()
+
+install_requires_ = ['numpy','nltk', 'jellyfish','pycountry']
```

Comment:
From my pov, it's easier to use the environment markers approach.

      install_requires=[
             ...,	
             'newspaper;python_version<"3.0"',		
             'newspaper3k;python_version>="3.0"',
             ...,	
       ],